README.md

```diff
@@ -35,28 +35,37 @@ not present any other legal issues.
 * OSI approved license are strongly preferred.
 * Mods do not need to be minetest-licensing compatible (meaning, (A)GPL3.0+ is fine).
 * Non-OSI licenses may be acceptable, but this is determined on a one-by-one case.
-* Pure Public Domain is not acceptable, however WTFPL, MIT or similar usually is acceptable.
+* Pure Public Domain is not acceptable, however ISC, MIT or similar usually is acceptable.
+* WTFPL is expressly discouraged and we ask that you relicense to another license
+  before submitting. There are several appropriate alternatives, such as MIT, ISC etc..
 
 The project admins reserve the right to discontinue, remove or refuse any project based
-on legality, content or other, at their discretion.
+on legality, content or other, at their discretion. However, removal requests for other
+than reasons other than non-acceptable content or licensing are unlikely to be granted.
```

Comment:
Duplicate `other than`.
Also, a different wording that I think works a bit better:
```
The project admins reserve the right to discontinue, remove or refuse any project based
on content, licensing, or other reasons, at their discretion. However, removal requests
for reasons other than unacceptable content or licensing are unlikely to be granted.
```